resource.html

```diff
@@ -0,0 +1,1003 @@
+                          </a>
+                        </div>
+                      </div>
+
+                      <div class="image aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                        <div data-cmp-lazythreshold="0" data-cmp-src="/content/leaderimpact/lang/en2/_jcr_content/content-parsys/container_428510884/container/container/container/container/image_512298232.coreimg{.width}.svg/1669130210337/insta.svg" data-asset="/content/dam/LeaderImpact/site-./assets/insta.svg" data-title="instagram" id="image-ae3671446d" class="cmp-image" itemscope="" itemtype="http://schema.org/ImageObject">
+                          <a href="#" class="cmp-image__link">
+                            <img src="./assets/insta.svg" class="cmp-image__image" itemprop="contentUrl" data-cmp-hook-image="image" alt="instagram">
+                          </a>
+                        </div>
+                      </div>
+
+                      <div class="image aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                        <div data-cmp-lazythreshold="0" data-cmp-src="/content/leaderimpact/lang/en2/_jcr_content/content-parsys/container_428510884/container/container/container/container/image_1887433809.coreimg{.width}.svg/1669130230195/yt.svg" data-asset="/content/dam/LeaderImpact/site-./assets/yt.svg" data-title="YouTube" id="image-46d1bf6845" class="cmp-image" itemscope="" itemtype="http://schema.org/ImageObject">
+                          <a href="#" class="cmp-image__link">
+                            <img src="./assets/yt.svg" class="cmp-image__image" itemprop="contentUrl" data-cmp-hook-image="image" alt="YouTube">
+                          </a>
+                        </div>
+                      </div>
+
+                      <div class="image aem-GridColumn aem-GridColumn--default--12 layout-grid">
+                        <div data-cmp-lazythreshold="0" data-cmp-src="/content/leaderimpact/lang/en2/_jcr_content/content-parsys/container_428510884/container/container/container/container/image_1801542987.coreimg{.width}.svg/1669130241057/linkedin.svg" data-asset="/content/dam/LeaderImpact/site-./assets/linkedin.svg" data-title="LinkedIn" id="image-bb85cd5a39" class="cmp-image" itemscope="" itemtype="http://schema.org/ImageObject">
+                          <a href="#" class="cmp-image__link">
+                            <img src="./assets/linkedin.svg" class="cmp-image__image" itemprop="contentUrl" data-cmp-hook-image="image" alt="LinkedIn">
+                          </a>
+                        </div>
+                      </div>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="container responsivegrid latest-post-banner aem-GridColumn aem-GridColumn--default--12" style="background:url(./assets/resource-banner.png);">
```

Comment:
@Prystal254 This background image needs to be on the child `.cmp-container` instead.